ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_creation.py

```diff
@@ -406,6 +406,29 @@ def test_ndindex(dtype_x_shape):
         assert index1 == index2
 
 
+@handle_test(
+    fn_tree="functional.ivy.experimental.polysub",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        max_num_dims=1,
+        min_value=-1000,
+        max_value=1000,
+    ),
+    test_with_out=st.just(True),
+    test_instance_method=st.just(False),
+)
+def test_polysub(*, dtype_and_x, dtype, test_flags, backend_fw, fn_name, on_device):
```

Comment:
`dtype` is not defined in `handle test`, which is why your test is currently failing.